modules/perception/camera/lib/traffic_light/detector/recognition/BUILD

```diff
@@ -28,8 +27,7 @@ cc_library(
         ":classify",
         "//cyber/common:file",
         "//modules/perception/camera/lib/interface",
-        # TODO(zhangxiao23@baidu.com): Rebuild perception inference module
-        # "//modules/perception/inference:inference_lib",
+        "//modules/perception/inference:inference_lib",
```

Comment:
It's usually sufficient to name a build target by its name only. For example, given source files `inference_util.{h,cpp}`, we generally use `cc_library(name = "inference_util", ...) ` to refer to them in `BUILD`. Rather than with a suffix  `_lib`.

When we write `#include "modules/perception/inference/utils/resize.h"` in C++, it generally means that our target will depend on `//modules/perception/inference/utils:resize`.

Reply:
@storypku I can't see anything wrong here. `inference_lib` is the original usage.

Reply:
> @storypku I can't see anything wrong here. `inference_lib` is the original usage.

It was not wrong or right. It's about Bazel Best Practice.  adding "_lib" doesn't improve anything.